.eslintrc.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
         "no-throw-literal": "error",
         "no-trailing-spaces": "error",
         "no-undef-init": "error",
-        "no-underscore-dangle": "error",
+        "no-underscore-dangle": "off",
```

Comment:
This setting is turned off to allow fields that can be mutated privately (`_foo: string`) but that are readonly publically (`get foo(): string { return this._foo; }`).